plugins/libplugin-pay.h

```diff
@@ -270,7 +280,7 @@ struct payment {
 
 	/* tal_arr of channel_hints we incrementally learn while performing
```

Comment:
This comment could be updated. It is no longer a `tal_arr` but a hash map from now on.

Reply:
Done